sdk/communication/azure-communication-rooms/tests/_shared/testcase.py

```diff
@@ -83,7 +83,7 @@ def setUp(self):
         super(CommunicationTestCase, self).setUp()
 
         if self.is_playback():
-            self.connection_str = "endpoint=https://rooms-ppe-us.ppe.communication.azure.net/;accesskey=J9gcDYLfopqKzHIKg7BI7+Qt/ZKTg0jeO/xvUF1JWxr8sHeA9Wq3DT+bjEIo3kRfjuj84CNm3s7B/zDrqkeLnA=="
+            self.connection_str = "endpoint=https://sanitized.ppe.communication.azure.net/;accesskey=J9gcDYLfopqKzHIKg7BI7+Qt/ZKTg0jeO/xvUF1JWxr8sHeA9Wq3DT+bjEIo3kRfjuj84CNm3s7B/zDrqkeLnA=="
```

Comment:
Please remove the access key from the string as well.
"endpoint=https://sanitized.communication.azure.com/;accesskey=fake==="